src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -428,28 +424,59 @@ private static Mock<ITextDocument> CreateDocumentMock(string fileName)
             var mockSnapshot = CreateMockTextSnapshot(1000, "some text");
             mockTextBuffer.Setup(x => x.CurrentSnapshot).Returns(mockSnapshot.Object);
 
+            return mockTextBuffer;
+        }
+
+        private static Mock<ITextDocument> CreateDocumentMock(string fileName, ITextBuffer textBuffer)
+        {
             // Create the document and associate the buffer with the it
             var mockTextDocument = new Mock<ITextDocument>();
             mockTextDocument.Setup(d => d.FilePath).Returns(fileName);
             mockTextDocument.Setup(d => d.Encoding).Returns(Encoding.UTF8);
 
-            mockTextDocument.Setup(x => x.TextBuffer).Returns(mockTextBuffer.Object);
+            mockTextDocument.Setup(x => x.TextBuffer).Returns(textBuffer);
 
             return mockTextDocument;
         }
 
-        private static Mock<ITextSnapshot> CreateMockTextSnapshot(int lineCount, string textToReturn)
+        private static Mock<ITextSnapshot> CreateMockTextSnapshot(int lineCount, string textToReturn, int versionNumber = 0)
         {
+            const int length = 9999;
+
+            var mockVersion = new Mock<ITextVersion>();
+            versionNumber = Math.Max(versionNumber, 1);
+            mockVersion.Setup(x => x.VersionNumber).Returns(versionNumber);
+            mockVersion.Setup(x => x.Length).Returns(length);
```

Comment:
I'll undo this change - we don't need a valid text version (left over from when I was trying to mock the data required by `SnapshotSpan.TranslateTo`).